main.tf

```diff
@@ -0,0 +1,146 @@
+data "aws_caller_identity" "current" {
+}
+
+data "aws_region" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+  region     = data.aws_region.current.name
+}
+
+resource "aws_security_group" "this" {
+  name_prefix = "${var.name}-scim-sg"
+  vpc_id      = var.vpc_id
+
+  tags = merge(
+    var.tags,
+    map(
+      "Name", "${var.name}-scim-sg"
```

Comment:
and if you set the names in locals you get to reference the same local variable in the tags name too